tests/test_upload_task.py

```
@@ -24,462 +22,123 @@
-        start_upload(sample_files_to_upload, "[[:File:Main.svg]]", "user", "pass", per_file_callback=callback)
-        
-        # Callback should be called for each file
-        assert callback.call_count == 2
-    
     @patch("src.web.upload_task.mwclient.Site")
-    def test_upload_login_required(self, mock_site_class, sample_files_to_upload):
-        """Test that login is attempted."""
+    @patch("src.web.upload_task.OAuthAuthentication")
+    def test_upload_collects_errors(
+        self,
+        mock_auth,
```

Ruff (0.14.1)

Unused method argument: `mock_auth`

(ARG002)